src/index.html

```diff
@@ -22,6 +22,433 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a
+        href="#"
+        class="container__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="container__logo"
+        />
+      </a>
+      <nav>
+        <ul class="container__nav-list">
+          <li class="container__link">
+            <a
+              href="#apple"
+              class="is-active container__nav-link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="container__link">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="container__link">
```